Planets and Systems/Mars.md

```diff
@@ -8,7 +8,7 @@ With the first human being standing on the Red Planet in 2052, to the nearly 7 b
 
 ### The Black Flag Movement
 
-An anarchist movement that started with the founding of the Confederacy of Man, the BLM is a violent terrorist group that operates out of Maza City and Catti. It's goals are for the disestablishment of Sol Gov, and the return of an "Open Citizen" state, though their practices and ways to obtain this are conflicting. Using bombings, assassinations, and smallscale guerilla warfare, the BLM has brought a crisis of sorts to the seat of Sol Gov. 
+An anarchist movement that started with the founding of the Confederacy of Man, the BLM is a violent terrorist group that operates out of Maza City and Catti. It's goals are for the disestablishment of Sol Gov, and the return of an "Open Citizen" state, though their practices and ways to obtain this are conflicting. Using bombings, assassinations, and smallscale guerilla warfare, the BFM has brought a crisis of sorts to the seat of Sol Gov. 
```

Comment:
I was literally just fixing the typo of BLM > BFM, but what should I replace the TC with?